sunpy/lightcurve/tests/test_stereo.py

```diff
@@ -0,0 +1,135 @@
+"""
+STEREO LightCurve Tests
+
+"""
+
+# This module was developed with funding from 
+# Google Summer of Code 2015
+# author - Ankit Kumar  <ankitkmr.iitk@gmail.com>
+
+from __future__ import absolute_import
+
+import pytest
+import os
+
+import sunpy.lightcurve
+from sunpy.time import TimeRange
+import sunpy.data.test
+
+filepath = sunpy.data.test.rootdir
+
+
+class TestLETLightCurve(object):
+
+    @pytest.mark.parametrize("data_file",[('Al_narrow_ahead.txt'),('Ar_summed_ahead.txt'),
+                 ('Fe_sectored_ahead_2015_001_level1_11.txt'), ('Al_summed_ahead.txt'), ('Ar_ahead_2006_318_level1_11.txt'), 
+                 ('Ar_summed_ahead_2015_1hr_level1_11.txt'), ('Fe_sectored_ahead_2015_01_10min_level1_11.txt'),
+                ('Al_summed_ahead_2007_01_10min_level1_11.txt'), ('Ar_narrow_ahead.txt'), ('CNO_lo_sectored_ahead_2015_1hr_level1_11.txt')])
+    @pytest.mark.online
+    def test_header(self, data_file):
+        """Test header parsing from file"""
+        lc = sunpy.lightcurve.LETLightCurve
+        assert (lc._parse_txt(os.path.join(filepath ,'let/' ,data_file))[0])[1][:14] in ['Flux for Bin 0', 'Column 6: LET '] 
+
+
+    @pytest.mark.parametrize("data_file",[('Al_narrow_ahead.txt'),('Ar_summed_ahead.txt'),
+                 ('Fe_sectored_ahead_2015_001_level1_11.txt'), ('Al_summed_ahead.txt'), ('Ar_ahead_2006_318_level1_11.txt'), 
+                 ('Ar_summed_ahead_2015_1hr_level1_11.txt'), ('Fe_sectored_ahead_2015_01_10min_level1_11.txt'),
+                ('Al_summed_ahead_2007_01_10min_level1_11.txt'), ('Ar_narrow_ahead.txt'), ('CNO_lo_sectored_ahead_2015_1hr_level1_11.txt')])
+    @pytest.mark.online
+    def test_data(self, data_file):
+        """Test for non empty data parsing from file"""
+        lc = sunpy.lightcurve.LETLightCurve
+        assert (lc._parse_txt(os.path.join(filepath ,'let/' ,data_file))[1]).empty == False
```

Comment:
here and few more places, the `os.path.join()` takes care of the `'/'` or `'\'` , so no need to put them in the middle directories.